### PR TITLE
More Generalizaton

### PR DESCRIPTION
**Pull request by ashishgulati71**

### CODE REVIEW

File: kitaab_bot.py [hunk 1]
```diff
@@ -103,8 +103,7 @@ def save_user_logs(msg, message):
 
 #Required Edit 3
 #Replace the below path with your path of log file
-
-    with open('/kitaab_bot/logs.txt', 'a') as log_file:
+    with open(LOG_PATH, 'a') as log_file:
```
Comment (singhsegv) [inline kitaab_bot.py]: use the file name logs.txt with LOG_PATH and recreate the request

**Review by singhsegv — CHANGES_REQUESTED** [review]

File: kitaab_bot.py (hunk 1 above)
Comment (ashishgulati71) [reply kitaab_bot.py]: done

**Comment by ashishgulati71** [comment]
done

**Comment by ashishgulati71** [comment]
editied

**Review by ashishgulati71 — COMMENTED** [review]
changed

**Review by singhsegv — APPROVED** [review]

**Pull request merged**